### PR TITLE
privacy policy

### PR DESCRIPTION
**Pull request by jduckles**
A basic privacy policy for our website.

### CODE REVIEW

**Comment by jduckles** [comment]
@gvwilson please review.

File: pages/privacy.md [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+information, but providing such information is completely optional.
+
+Voting Members will need to provide us with email contact information during
+their enrollment. However, that information is never made generally available
+outside of the Foundation, neither on this website nor anywhere else public.
+
+## 2. What do we use this information for?
+
+**Basic usage information.** We use basic usage information to protect against
+**unauthorized intrusion into SCF-controlled servers, as well as monitor and
```
Comment (gvwilson) [inline pages/privacy.md]: What are the asterisks at the start of these lines for?

File: pages/privacy.md [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+**improve the availability and performance of SCF-controlled websites. We may
+**also use information collected from you to personalize your interactions with
+**SCF-controlled websites.
+
+**Personally identifiable information.** We use contact information to
+**communicate with members or with others that have accounts with the SCF, to
```
Comment (gvwilson) [inline pages/privacy.md]: As above - looks like indent asterisks gone mad?

File: pages/privacy.md [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+**measures are in place in safeguarding your personal data.
+
+## 3. Disclosure of personal information
+
+We will not disclose any information we collect to any third party unless it is:
+
+1. Specifically authorized by you; or 2. Necessary to perform the functions
+requested by you (and only to the extent absolutely necessary to perform the
+function); or 3. Required to check for fraud or other misuse of SCF resources;
+or 5. Required by law.
```
Comment (gvwilson) [inline pages/privacy.md]: 1, 2, 3, 5 - maybe just break this out into an enumerated list and let Markdown do the numbering?

File: pages/privacy.md [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+
+In particular, we may release the information we collect to third parties when
+we believe it is appropriate to comply with the law, to enforce our legal
+rights, to protect the rights and safety of others, or to assist with industry
+efforts to control fraud, spam or other undesirable conduct.
+
+If the SCF contracts with a third party to provide a particular service, we may
```
Comment (gvwilson) [inline pages/privacy.md:92]: Um... add a clause here saying specifically that third parties will be required to _only_ use the information for the purposes contracted, and will not be allowed to pass it on to anyone else without our prior explicit consent?

Comment (jduckles) [inline pages/privacy.md:92]: I think that is covered by "and is permitted to use the information only for the
purpose of providing services to us"

File: pages/privacy.md (hunk 1 above)
Comment (jduckles) [inline pages/privacy.md]: Yeah, my editor was err helping me...by you know, not helping me.

**Pull request merged**